packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -43,7 +43,10 @@ export async function tailLogGroup(
         }
         const session = new LiveTailSession(liveTailSessionConfig)
         if (registry.has(uriToKey(session.uri))) {
-            await prepareDocument(session)
+            await vscode.window.showTextDocument(session.uri, { preview: false })
+            void vscode.window.showInformationMessage(
+                `Switching editor to an existing session. Found running session that matches request.`
```

Comment:
I feel like these two sentences mean the same thing, we can we simplify them? Otherwise the PR LGTM